### DISCUSSION TITLE
Did anything change when the query has 'ORDER BY DESC'?

### DISCUSSION THREAD

**Discussion by xariskap** [jackc/pgx#2018] [General]
I need to do a few timing measurements, and over the past days the timings i get (~1.7 sec) are different that those of 2 weeks before (~0.09 sec).
Even with empty `for rows.Next()` the time is constantly 1.7sec

**Comment by jackc** [comment]
An `order by desc` clause should not change anything with regard to pgx. It may cause PostgreSQL to execute with a different plan You could try running your queries in psql to remove pgx from the equation (you might want to `PREPARE` and `EXECUTE` the statements in psql to more closely match how pgx executes queries).
